litmus/director/cluster-connect-check/test_vars.yml

```diff
@@ -2,3 +2,5 @@ test_name: cluster-connect-check
 director_client_namespace: "{{ lookup('env','DIRECTOR_CLIENT_NAMESPACE') }}"
 client_components: ['cortex-agent','dmaas-agent','fluentd-aggregator','fluentd-forwarder','kube-state-metrics','maya-io-agent','openebs-manager','status-agent','upgrade-controller','velero','weave-scope-agent','weave-scope-app','weave-scope-cluster-agent']
 director_ip: "{{ lookup('env','DIRECTOR_IP') }}"
+cluster_name: "{{ lookup('env','CLUSTER_NAME') }}"
+provider: "{{ lookup('env','PROVIDER') }}"
```

Comment:
@amitbhatt818 `cluster_name` and `provider` are not being passed in env right now, we need to pass it as env to get it here.